lib/govuk_schemas/random_example.rb

```diff
@@ -40,93 +40,70 @@ def self.for_schema(schema_key_value)
     # Example:
     #
     #      random = GovukSchemas::RandomExample.for_schema("detailed_guide", schema_type: "frontend")
-    #      random.customise_and_validate({base_path: "/foo"}, ["withdrawn_notice"])
+    #      random.payload
     #      # => {"base_path"=>"/e42dd28e", "title"=>"dolor est...", "publishing_app"=>"elit"...}
     #
-    # @param [Hash] hash The hash to merge the random content with
-    # @param [Array] array The array containing fields to exclude
+    #      random.payload do |payload|
+    #        payload['base_path'] = "Test base path"
+    #      end
+    #      # => {"base_path"=>"Test base path", "title"=>"dolor est...", "publishing_app"=>"elit"...}
+    #
+    # @param [Block] hash which can be modified before being validated
```

Comment:
The wording is a bit strange here, it sounds a bit like the hash is the parameter.

Maybe it should say something like "an optional block that receives the generated content item and can modify it before it gets validated"

Reply:
updated